docs/content/0.1.0/the-hss-interface.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
@CircArgs it's probably worth you taking a look at this attempt to describe how the semantic search fits into the bigger picture. Feel free to merge and then open a PR that changes most of this.